tests/e2e/test_kb_builder_pipeline_e2e.py

```diff
@@ -97,16 +97,25 @@ def kg_builder_pipeline(
     pipe = Pipeline()
     # define the components
     pipe.add_component(
-        "splitter",
         text_splitter,
+        "splitter",
+    )
+    pipe.add_component(
+        chunk_embedder,
+        "embedder",
+    )
+    pipe.add_component(
+        schema_builder,
+        "schema",
     )
-    pipe.add_component("embedder", chunk_embedder)
-    pipe.add_component("schema", schema_builder)
     pipe.add_component(
-        "extractor",
         entity_relation_extractor,
+        "extractor",
```

Comment:
Is there meant to be a difference between `pipeline_branch` and `pipeline_aggregation`?

Reply:
Hum, yes, the idea was to have different connections:
- branching: a -> b, a -> c
- aggregation: a -> c, b -> c
I'll take a look as I'm planing to revisit a bit these tests (TBD)